server.js

```diff
@@ -1,5 +1,3 @@
-process.env.UV_THREADPOOL_SIZE = 100;
```

Comment:
In spec server, it is unnecessary. But I have added it.
Is it  the node-sass issue?